chat-plugins/chat-monitor.js

```diff
@@ -114,6 +129,17 @@ let chatfilter = function (message, user, room) {
 			return false;
 		}
 	}
+	for (let i = 0; i < filterWords.shorteners.length; i++) {
+		let [regex] = filterWords.shorteners[i];
+		if (typeof regex === 'string') continue; // Failsafe to appease typescript.
```

Comment:
This is a fail-dangerous, not a failsafe. You should `throw` if this is a situation you think will never happen.

Reply:
```suggestion
		if (typeof regex === 'string') throw new Error(`shortener filters should not have strings`);
```